src/backend/joanie/core/api/admin.py

```diff
@@ -30,6 +30,14 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = models.Organization.objects.all()
     filterset_class = filters.OrganizationAdminFilterSet
 
+    def get_permissions(self):
+        """
+        Instantiates and returns the list of permissions that this view requires.
+        """
+        if self.action in ["list", "retrieve"]:
+            return [permissions.IsAdminUser()]
+        return super().get_permissions()
+
```

Comment:
I do not understand why you need to change this logic 🤔

Reply:
Maybe I misunderstood, but I think this is the right PR.